packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -99,26 +88,31 @@ const TeamPromptMeeting = (props: Props) => {
 
   const phase = getPhaseByTypename(phases, 'TeamPromptResponsesPhase')
   const stages = useMemo(() => {
-    const allStages = phase.stages
-      .map((stage) => {
-        return {
-          ...stage,
-          key: stage.id
-        }
-      })
-      .filter(isNotNull)
-
-    // Find the viewer's card and put it at the beginning.
+    const allStages = phase.stages.filter(isNotNull)
+
+    const nonViewerStages = allStages.filter((stage) => stage.teamMember.userId !== viewerId)
+    const orderedNonEmptyStages = nonViewerStages
+      .filter((stage) => !!stage.response?.plaintextContent)
+      .sort((stageA, stageB) =>
+        sortByISO8601Date(stageA.response!.createdAt, stageB.response!.createdAt)
```

Comment:
+1 Wouldn't it be more intuitive to order by `updatedAt`?

Reply:
Per the [issue](https://github.com/ParabolInc/parabol/issues/6561), we decided to optimize for fewer layout changes in non-empty cards to avoid the case where a card moves while a user is reading it.  So similar to not pushing a new card to the front of the list, we probably don't want to be moving updated cards to the end of the list (thus shifting all non-empty cards up one space).

We'll probably revisit this ordering if there are tensions.